Makefile

```diff
@@ -128,15 +128,15 @@ recompute-licenses: ## Recompute the licenses.csv only if needed (gomod was chan
 
 licenses-up-to-date:
 	@if [ "$(GOMOD_SHA)" != "$(GOMOD_LICENSES_SHA)" ]; then \
-	echo "licenses.csv needs ot be recalculated: run 'make licenses.csv'"; exit 1; else\
-	echo "licenses.csv is OK! (up to date)"; fi
+	echo "licenses.csv needs to be recalculated: git rebase AND run 'make licenses.csv'"; exit 1; \
```

Comment:
This tripped me for quite a while.

The check will fail if you need to rebase as well, even thought locally the `licenses.csv` seems up to date, if the diff includes a rebase, the go.mod will be different and thus the `licenses.csv` need to be recomputed.

While I was trying to fix this I also removed our dependency on shasum and use git SHAs directly instead.